css/layout.css

```diff
@@ -1,126 +1,125 @@
 body {
-  display: inline-block;

```

Comment:
It looks like you are using flexbox. If so, make sure to put 'display { flex };'